.github/workflows/playwright.yml

```diff
@@ -24,8 +24,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        shardIndex: [1, 2, 3, 4, 5, 6, 7, 8]
-        shardTotal: [8]
+        shardIndex: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        shardTotal: [10]
```

Comment:
Added 2 more here because of the parallel change causing tests to run a bit slower. At some point, you'll get diminishing returns having more of them, but let's adjust as needed.